README.md

```diff
@@ -30,22 +30,6 @@ git clone https://github.com/kubernetes/website.git
 cd website
 ```
 
-The Kubernetes website uses the [Docsy Hugo theme](https://github.com/google/docsy#readme). Even if you plan to run the website in a container, we strongly recommend pulling in the submodule and other development dependencies by running the following:
-
-### Windows
-
-```powershell
-# fetch submodule dependencies
-git submodule update --init --recursive --depth 1
-```
-
-### Linux / other Unix
-
-```bash
-# fetch submodule dependencies
-make module-init
-```
-
 ## Running the website using a container
 
 To build the site in a container, run the following:
```

Comment:
we may want to update these instructions.

Currently, from a fresh repo, starting with `make module-init` fails with
```
Initializing submodules...
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
fatal: remote error: upload-pack: not our ref 096a94d6e7d0d04e41356f21233498c3c4e31699
fatal: Fetched in submodule path 'api-ref-generator', but it did not contain 096a94d6e7d0d04e41356f21233498c3c4e31699. Direct fetching of that commit failed.
make: *** [module-init] Error 128
```

I can't get past this for running this with a container. I thought maybe I need to do an npm install or npm ci first, but neither of those work from a clean pull either.

Reply:
I'm looking into my own setup here as I'm not sure this PR should cause this error.